frontend/src/pages/hardwareProfiles/useWatchHardwareProfile.ts

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { useK8sWatchResource } from '@openshift/dynamic-plugin-sdk-utils';
+import { HardwareProfileKind } from '#~/k8sTypes';
+import { groupVersionKind, HardwareProfileModel } from '#~/api';
+import { CustomWatchK8sResult } from '#~/types';
+
+const useWatchHardwareProfile = (
```

Comment:
i would just use a regular GET and not watch here. was there a reason to use watch

Reply:
also u should then use useFetch

Reply:
wouldn't the labels just show up on page load? what is happening in the background here